Content.Client/Atmos/UI/GasCanisterWindow.xaml.cs

```diff
@@ -82,9 +107,12 @@ public void SetReleasePressureRange(float min, float max)
 
         public void SetReleasePressure(float pressure)
         {
+            if (MathHelper.CloseTo(pressure, ReleasePressure.Value))
+                return;
+
             if(!ReleasePressureSlider.Grabbed)
                 ReleasePressureSlider.SetValueWithoutEvent(pressure);
-            ReleasePressureLabel.Text = Loc.GetString("comp-gas-canister-ui-pressure", ("pressure", Math.Round(pressure)));
```

Comment:
where is the label being set at now?

Reply:
I just removed the label in favour of the FloatSpinBox. It shows the same info (except for the "kPa").

Reply:
i think i'd prefer having the label, the `kPa` is useful i think

Reply:
IMO the spinbox is better because it makes it much easier to do things like set release pressure to specific values. E.g., if you wanted to release 20kpa oxygen canister in a room, and then afterwards open a nitrogen up to 100, its quite finicky to do that with just the slider.

and slider+spinbox+label seems overkill. I could just replace the "Release Pressure:" with "Release Pressure (kPa):"

Reply:
sure thats fine